js/glotdict-functions.js

```diff
@@ -459,12 +459,11 @@ function gd_add_evt_listener( event_name, target_selector, function_to_call ) {
  * @returns {void}
  */
 function gd_copy_to_clipboard( copy_text ) {
-	const elem = document.createElement( 'textarea' );
-	elem.value = copy_text;
-	document.body.appendChild( elem );
-	elem.select();
-	document.execCommand( 'copy' );
-	document.body.removeChild( elem );
+	navigator.permissions.query({name: "clipboard-write"}).then(result => {
```

Comment:
@webaxones, @Mte90, @vlad-timotei,
So if I understand all of your edits combined, I can just remove the conditional permissions check and just have the copy function inside `gd_copy_to_clipboard( copy_text )` like so:

`function gd_copy_to_clipboard( copy_text ) {`
`       navigator.clipboard.writeText( copy_text );`
`}`

I'm not very familiar with jQuery so forgive me if I missed something important in your PR, @Mte90.

And thanks everyone for the welcome! I'm delighted to help out if I can!  I'm still new to contributing on github so I'll probably ask some beginner questions fyi harhar

Reply:
Hi Dillon, all you can do... you can do :D

Reply:
That's very well said, thank you!